packages/atomic/src/components/commerce/atomic-commerce-no-products/atomic-commerce-no-products.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  SearchSummary,
+  ListingSummary,
+  buildSearchSummary,
+  buildListingSummary,
+  SearchSummaryState,
+  ListingSummaryState,
+} from '@coveo/headless/commerce';
+import {Component, State, h} from '@stencil/core';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {NoItemsContainer} from '../../common/no-items/container';
+import {NoItemsGuard} from '../../common/no-items/guard';
+import {MagnifyingGlass} from '../../common/no-items/magnifying-glass';
+import {NoItems} from '../../common/no-items/no-items';
+import {SearchTips} from '../../common/no-items/tips';
+import {CommerceBindings} from '../atomic-commerce-interface/atomic-commerce-interface';
+
+/**
+ * @internal
+ *
+ * The `atomic-no-products` component displays search tips when there are no products. Any additional content slotted inside of its element will be displayed as well.
+ *
+ * @part no-results - The text indicating that no products were found for the search.
```

Comment:
Let's start using the `@slot` annotation for anything that looks like a slot.
So here
`@slot default - lorem ipsum`